jax/_src/lax/lax.py

```diff
@@ -2171,19 +2171,21 @@ def _compare_lower_mhlo(direction: str, ctx, x, y):
 ad.defjvp_zero(ne_p)
 mlir.register_lowering(ne_p, partial(_compare_lower_mhlo, "NE"))
 
-ge_p = naryop(_fixed_dtype(np.bool_), [_any, _any], 'ge')
+_ordered = _int | _float | _bool
```

Comment:
Perhaps define this above, where `_num`, `_any`, etc. are defined

Reply:
Done.